llvm/include/llvm/ExecutionEngine/Orc/IndirectionUtils.h

```diff
@@ -77,7 +77,7 @@ class TrampolinePool {
     std::lock_guard<std::mutex> Lock(TPMutex);
     if (AvailableTrampolines.empty()) {
       if (auto Err = grow())
-        return std::move(Err);
+        return Err;
```

Comment:
There is still here the mismatch in the type of Err and the type of the return for the method that led to this error I believe: https://lab.llvm.org/buildbot/#/builders/61/builds/57655